### PR TITLE
feat: Added Software Testing YouTube Videos Details in testing.json File

### LINKED ISSUE #1345: [Add]: Add Testing Course for Learning Automation Testing related providing by EDUREKA
### Site URL

https://www.youtube.com/watch?v=9p6NNapsUvQ

### Description of the above provided site

- Add Testing Course for Learning Automation Testing related providing by EDUREKA.
- This Course is Updated in this Year and this Course maintains Current Tech Stacks

### Category

Youtube

### Sub Category

testing

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by NarendraKoya999**
## Fixes Issue:
Closes #1345 #1346 #1347 

## Changes proposed

- Added `Software Testing Course YouTube Video` details in `testing.js File in youtube Folder`.
- YouTube channels named as `Software Testing Mentor, Automation Step by Step and EDUREKA(Testing)` 
## Note to reviewers

- ` I have linked All related Testing related Issues raised by Me and also combined together with this PR `
- ` Please Notify Them` @rupali-codes

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Hi, can you please make the description a little longer?

**Comment by NarendraKoya999** [comment]
> Hi, can you please make the description a little longer?

Sure, I will do that?

**Review by CBID2 — APPROVED** [review]
Made some tweaks to your descriptions. Other than that, well done! :)

**Comment by NarendraKoya999** [comment]
> Made some tweaks to your descriptions. Other than that, well done! :)



> Made some tweaks to your descriptions. Other than that, well done! :)

If you don't mind Please add level 2 OR Level 3 Label to This PR. Because of If one resource is added, You can given level 1. But, I added 6 Resources in One PR. Please Taken my Humble Request @CBID2

**Comment by CBID2** [comment]
> > Made some tweaks to your descriptions. Other than that, well done! :)
> 
> 
> 
> 
> 
> 
> 
> > Made some tweaks to your descriptions. Other than that, well done! :)
> 
> 
> 
> If you don't mind Please add level 2 OR Level 3 Label to This PR. Because of If one resource is added, You can given level 1. But, I added 6 Resources in One PR. Please Taken my Humble Request @CBID2  

Sorry @NarendraKoya999. The rules state that adding links are level one, so we have to adhere to that.

**Comment by NarendraKoya999** [comment]
> > > Made some tweaks to your descriptions. Other than that, well done! :)
> > 
> > 
> > > Made some tweaks to your descriptions. Other than that, well done! :)
> > 
> > 
> > If you don't mind Please add level 2 OR Level 3 Label to This PR. Because of If one resource is added, You can given level 1. But, I added 6 Resources in One PR. Please Taken my Humble Request @CBID2
> 
> Sorry @NarendraKoya999. The rules state that adding links are level one, so we have to adhere to that.

Ok. No Problem @CBID2

File: database/youtube/testing.json [hunk 1]
```diff
@@ -1,26 +1,50 @@
 [
   {
     "name": "SoftwaretestingbyMKT",
-    "description": "This channel is for everyone who wants to learn Manual testing and Automation Testing for FREE.The reason he have made this YouTube channel is to educate people who want to learn in easy language. I have a good theory and practical knowledge on testing.",
+    "description": "This channel is for everyone who wants to learn Manual testing and Automation Testing for FREE. The reason he has made this channel is to educate people who want to learn an easy language. I have a good theory and practical knowledge on testing.",
```
Comment (rupali-codes) [inline database/youtube/testing.json:4]:
```suggestion
    "description": "This channel is for everyone who wants to learn Manual testing and Automation Testing for FREE. ",
```

**Review by rupali-codes — APPROVED** [review]

**Comment by NarendraKoya999** [comment]
Hi @rupali-codes . Please Look into 4 unassigned Pending Issues as soon as possible

**Pull request merged**